lib/cc/yaml/nodes/mapping.rb

```diff
@@ -75,10 +75,6 @@ def set_warnings(key)
 
       def []=(key, value)
         if mapped_key = mapped_key(key)
-          unless value.is_a? Node
-            node  = subnode_for(mapped_key)
-            value = node if Parser::Ruby.new(value).parse(node)
```

Comment:
@pbrisbin Yes that sounds good. Seems like a good candidate for concurrent fixing.